tips/TIP-0049/tip-0049.md

```diff
@@ -0,0 +1,567 @@
+            <td>Decay Balancing Constant is an integer approximation calculated based on chosen DecayBalancingConstantExponent.</td>
+          </tr>
+          <tr>
+            <td>Pool Coefficient Exponent</td>
+            <td>uint8</td>
+            <td>Pool Coefficient Exponent is the exponent used for shifting operation in the pool rewards calculations.</td>
+          </tr>
+        </table>
+      </details>
+    </td>
+  </tr>
+  <tr>
+    <td>Target Committee Size</td>
+    <td>uint8</td>
+    <td>Target Committee Size defines the target size of the committee. If there's fewer candidates the actual committee size could be smaller in a given epoch.</td>
+  </tr>
+</table>
+
+When referring to Protocol Parameters contained in subschemas in other TIPs, one may use namespacing with a `::`
+separator for disambiguation. For instance, `Work Score Parameters::Block` refers to the `Block` parameter inside the
+`Work Score Parameters`.
+
+## Protocol Parameter Hash
+
+The Protocol Parameter Hash is used in signaling protocol parameter upgrades and is a field in the Validation Block.
+
+The Protocol Parameter Hash is computed as follows:
+
+- Let `Protocol Parameter Bytes` be the serialized protocol parameter.
+- Let the resulting hash be the BLAKE2b-256 hash over `Protocol Parameter Bytes`.
+
+### Protocol parameter example
+
+The example is a protocol parameter filled with random values:
+
+```
+{
+  "type": 0,
+  "version": 3,
+  "networkName": "TestJungle",
+  "bech32Hrp": "tgl",
+  "storageScoreParameters": {
+    "storageCost": "0",
+    "factorData": 0,
+    "offsetOutputOverhead": "0",
+    "offsetEd25519BlockIssuerKey": "0",
+    "offsetStakingFeature": "0",
+    "offsetDelegation": "0"
+  },
+  "workScoreParameters": {
+    "dataByte": 0,
+    "block": 1,
+    "input": 0,
+    "contextInput": 0,
+    "output": 0,
+    "nativeToken": 0,
+    "staking": 0,
+    "blockIssuer": 0,
+    "allotment": 0,
+    "signatureEd25519": 0
+  },
+  "manaParameters": {
+    "bitsCount": 63,
+    "generationRate": 1,
+    "generationRateExponent": 17,
+    "decayFactors": [
+      10,
+      20
+    ],
+    "decayFactorsExponent": 32,
+    "decayFactorEpochsSum": 2420916375,
+    "decayFactorEpochsSumExponent": 21
+  },
+  "tokenSupply": "2779530283277761",
+  "genesisSlot": 0,
+  "genesisUnixTimestamp": "1695275822",
+  "slotDurationInSeconds": 10,
+  "slotsPerEpochExponent": 13,
+  "stakingUnbondingPeriod": 10,
+  "validationBlocksPerSlot": 10,
+  "punishmentEpochs": 10,
+  "livenessThresholdLowerBound": 15,
+  "livenessThresholdUpperBound": 30,
+  "minCommittableAge": 10,
+  "maxCommittableAge": 20,
+  "epochNearingThreshold": 24,
+  "congestionControlParameters": {
+    "minReferenceManaCost": "1",
+    "increase": "0",
+    "decrease": "0",
+    "increaseThreshold": 800000,
+    "decreaseThreshold": 500000,
+    "schedulerRate": 100000,
+    "maxBufferSize": 1000,
+    "maxValidationBufferSize": 100
+  },
+  "versionSignaling": {
```

Comment:
This is the only parameters struct that doesn't have a `Parameters` suffix. Maybe we should harmonise for consistency.